helm/promtail/templates/cilium-networkpolicy.yaml

```diff
@@ -34,5 +34,14 @@ spec:
         rules:
           dns:
           - matchPattern: '*'
+    - toEndpoints:
+      - matchLabels:
+          app.kubernetes.io/name: loki
+      toPorts:
+      - ports:
+        - port: "80"
+          protocol: ANY
+        - port: "9095"
```

Comment:
We might not need 9095 since we point to the loki gateway service.
```
loki-gateway                ClusterIP   172.31.249.121   <none>        80/TCP              240d
```

Reply:
ok that works for me